database/models/Card.js

```diff
@@ -1,7 +1,7 @@
 const bookshelf = require('./bookshelf');
 
 class Card extends bookshelf.Model {
-  get teableName() {
+  get tableName() {
```

Comment:
These are good fixes, I called these out in another PR. Generally best practice to fix on the original branch it was identified on 👍